checks/E4D1B4.yaml

```diff
@@ -0,0 +1,70 @@
+id: "E4D1B4"
+name: SAPHanaTopology monitor interval has correct value
+group: Pacemaker
+description: |
+  SAPHanaTopology monitor interval has correct value
+remediation: |
+  ## Abstract
+  SAPHanaTopology monitor interval should have recommended value in cluster configuration. 
+
+  ## Remediation
+
+  Edit the crm configuration and change the value for this parameter to 50.
+
+  ## References
+  AZURE:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-1.11.7.10.4
+    - https://learn.microsoft.com/en-us/azure/sap/workloads/sap-hana-high-availability?tabs=lb-portal%2Csaphanasr-angi#create-sap-hana-cluster-resources
+
+  AWS:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-1.11.7.10.4
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-perfopt-15/index.html#id-saphanatopology
+
+  GCP:
+
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-scaleout-config-sles#create_the_saphanatopology_primitive_resource
+    - https://cloud.google.com/solutions/sap/docs/sap-hana-ha-config-sles#create_the_saphanatopology_primitive_resource
+
+  SUSE / KVM:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-1.11.7.10.4
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-perfopt-15/index.html#id-saphanatopology    
+
+  Nutanix:
+
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-scaleout-perfopt-15/index.html#id-1.11.7.10.4
+    - https://documentation.suse.com/sbp/sap-15/html/SLES4SAP-hana-angi-perfopt-15/index.html#id-saphanatopology    
+
+severity: warning
+metadata:
+  target_type: cluster
+  cluster_type: 
+    - hana_scale_up
+    - hana_scale_out
+  architecture_type: angi
+
+facts:
+  - name: topo_monitor
+    gatherer: cibadmin
+    argument: cib.configuration.resources
+
+values:
+  - name: topo_interval
+    default: 50      
+
+expectations:
+  - name: expectations_topo_monitor_interval
+    expect: |
+          let hana_ct = facts.topo_monitor.clone;
```

Comment:
we don't need indentations after `let`. we should use the same level for the next `for`.
PD: you don't even need the `let`, but if it adds meaningfulness...